Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -91,6 +94,11 @@ public static FluentLayout WithSameHeight(this UIView view, UIView previous)
             return view.Height().EqualTo().HeightOf(previous);
         }
 
+        public static FluentLayout WidthEqualHeight(this UIView view, UIView previous, nfloat? margin=null, nfloat? scale = null)
+        {
+            return view.Width().EqualTo().HeightOf(previous).Minus(2*margin.GetValueOrDefault(DefaultMargin)).WithMultiplier(scale.GetValueOrDefault(DefaultScale));
+        }
```

Comment:
The `2*` here is interesting... I understand it (I think)... nice touch :) (makes the margin parameter consistent with other methods like `FullWidthOf`  :+1: LIKE)